sample.conf

```diff
@@ -453,6 +453,27 @@
 ## The LDAP filter for the search
 #filter="(&(objectClass=inetOrgPerson)(!(loginShell=/bin/false)))"
 
+## The method used for LDAP bind
+## The possible values are:
+## "none": Do not use any LDAP bind. This is useful if the authentication is handled by the os config.
+## "simple": LDAP simple bind. In this case a dn must be set. The password will be requested via a GUI dialog and stored encrypted.
+## "gssapi": LDAP SASL bind per GSSAPI. Which other options (see below) must be set depends on the server configuration.
+##
+## default: "none"
+## (optional)
+#bindMethod="none"
+
+## Whether to use SSL/TLS for the LDAP bind
+## (optional, default: true)
+# useSSL=true
```

Comment:
@mbgonicus how can we set the CA certificate if required?

Reply:
Ups, forgot that, sorry. I added the documentation for it.